lib/app/modules/uscore_v3.0.1/pediatric_bmi_for_age_sequence.rb

```diff
@@ -76,22 +74,26 @@ def validate_resource_item(resource, property, value)
           versions :r4
         end
 
-        search_params = { patient: @instance.patient_id, code: '59576-9' }
+        code_val = ['59576-9']
+        code_val.each do |val|
```

Comment:
Could this be `codes.each do |code|` instead?

Reply:
It's the same issue as the question below. If the parameter is  'class', then we run into issues. This isn't an issue in US core, but I figured the awkward naming was worth making it more generic.